### ISSUE TITLE
Delete in-tree support for Nvidia GPUs

### ISSUE THREAD

**Issue by vishh** [kubernetes/kubernetes#54012]
Nvidia GPU support is currently available via the kubelet directly. This support has been deprecated in favor of device plugins. Since we don't have a device plugin for all distros, deleting the old support isn't advisable just yet. 
I'm hoping we can delete the old implementation which works by setting `Accelerators=true` feature gate in v1.10 release cycle.

Depends on #54011

#53497 for tracking

**Comment by dims** [comment]
per comment from @mindprince looks like we need to move this milestone to v1.11 (https://github.com/kubernetes/kubernetes/issues/54101#issuecomment-359127274)

**Comment by jiayingz** [comment]
+1 for moving it to v1.11. We will announce the deprecation in v1.10.

**Comment by jberkus** [comment]
please add kind/ and priority/ labels to this issue for 1.10. Thanks.

**Comment by jberkus** [comment]
since #54011 appears to be delayed, please make a decision on this issue ASAP.  are we announcing deprecation if the new GPU support isn't ready?

**Comment by jiayingz** [comment]
We mark the feature as deprecated in 1.10. Will remove the in-tree code in 1.11. Please feel free to move the milestone to 1.11.

**Comment by jberkus** [comment]
OK, is there a separate issue (maybe a Doc issue) for announcing deprecation?

**Comment by mindprince** [comment]
The GPU doc (https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/) was already updated to say:
> The Accelerators feature gate and alpha.kubernetes.io/nvidia-gpu resource works on 1.8 and 1.9 as well. It will be deprecated in 1.10 and removed in 1.11.

This should also appear in 1.10 release notes: https://github.com/kubernetes/kubernetes/pull/57384

**Comment by mindprince** [comment]
@jberkus Can you please move this to 1.11 milestone (that's when we will delete the code).

/assign

**Comment by jberkus** [comment]
@mindprince a sig lead needs to change the milestone.  Automation will remove 1.10 in 7 hours, but it won't put 1.11 in.
